services/listen_dnsport.c

```diff
@@ -1288,7 +1708,9 @@ set_recvpktinfo(int s, int family)
  * @param dnscrypt_port: dnscrypt service port number
  * @param dscp: DSCP to use.
  * @param quic_port: dns over quic port number.
- * @param http_notls_downstream: if no tls is used for https downstream.
+ * @param coap_port: dns over coap port number.
+ * @param coaps_port: dns over coaps port number.
+ * @param coap_context: a pre-created CoAP context for dns over coap.
```

Comment:
The 'https_notls_downstream' variable documentation dropped off, this documents it.
```suggestion
 * @param coap_context: a pre-created CoAP context for dns over coap.
 * @param http_notls_downstream: if no tls is used for https downstream.
```